<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copyright 2022. Sanghyeok Kong All rights reserved.</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>142</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>맑은 고딕 Semilight</family>
       </font>
      </property>
      <property name="text">
       <string>엑셀 파일 가져오기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <family>맑은 고딕 Semilight</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>CSV 파일 가져오기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <family>맑은 고딕 Semilight</family>
       </font>
      </property>
      <property name="text">
       <string>저장위치 설정</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>맑은 고딕 Semilight</family>
       </font>
      </property>
      <property name="text">
       <string>시작</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="font">
       <font>
        <family>맑은 고딕 Semilight</family>
       </font>
      </property>
      <property name="text">
       <string>서버에 저장</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
